charts/optimize-live/values.yaml

```diff
@@ -26,6 +26,17 @@ component:
       repository: grafana/grafana-oss
       pullPolicy: IfNotPresent
       tag: 8.5.4
+  resources:
```

Comment:
this approach is minimizes changes to existing values.yaml for backward compatibility.  however there are a few changes that seem preferable if we don't mind refactoring it a bit:

1. remove `controller` from the `component` section and simply have `image:` section at root level (which would be consistent with the rest of the `controller` config (i.e. resources, replicaCount, tolerations, affinity, etc)
2. remove `pullPolicy` from component `image` section, or support passing it through for components, as it currently doesn't appear to be used (other than for controller).
3. refactor to be of the following structure or similar (having the component name under the components key seems more natural & more concise as number of settings increases - ie. adding things like `priorityClassName`):
```yaml
components:
  tsdb:
    image: ...
    resources: ...
  applier:
...
```